### ISSUE TITLE
Extend hardware selection based on presence of a battery

### ISSUE THREAD

**Issue by kurik** [teemtee/tmt#2862]
Hi there,

on [PCP](https://github.com/performancecopilot/pcp) project we are gathering some metrics from batteries (used typically on laptops). To enable automated testing I would like to have the opportunity to select relevant hardware to run tests on. Such a hardware needs to have a battery and a battery controller present and communicating with the system.

On Beaker I am using `Device/Description contains battery` filter, however it is not fully reliable. For the reliable selection I need to run `lshw | grep -i battery` on booted system and check what the `lshw` returns.

**Comment by happz** [comment]
No Beaker job XML filter at hand, we would need to experiment with Beaker to create one.

**Comment by psss** [comment]
Seems like a nice enhancement, but blocked on missing features. Shall we add it to the backlog as a nice-to-have. Maybe, once the time comes also for this one?
